Add reference of DataFrame.rename_axis and Series.rename_axis to api.rst

shawnheide opened the pull request:
- [x] closes #13642
- [ ] tests added / passed
- [ ] passes `git diff upstream/master | flake8 --diff`
- [ ] whatsnew entry

sinhrks commented on doc/source/api.rst:
```diff
@@ -889,6 +889,7 @@ Reindexing / Selection / Label manipulation
    DataFrame.reindex_axis
    DataFrame.reindex_like
    DataFrame.rename
+   DataFrame.rename_axis
```
thx. can u also add to `Series`?

shawnheide committed:
DOC: Add reference of DataFrame.rename_axis and Series.rename_axis to api.rst

shawnheide commented:
@sinhrks I added the commit for Series as requested and squashed the commits together. This is my first PR, so feel free to let me know if there's a better way to do this.

TomAugspurger commented:
Thanks. Congrats on your first PR to pandas 🎉 

The changes should show up [here](http://pandas-docs.github.io/pandas-docs-travis/) a bit after [this](https://travis-ci.org/pydata/pandas/builds/145343146) Travis run finishes.
